conda.yaml

```diff
@@ -80,10 +80,14 @@ subpackages:
         - py${{range.key}}-ruamel-yaml
         - py${{range.key}}-tqdm
     pipeline:
+      - runs: |
+          # We don't use this dir until further down, but py/pip-build-install
+          # will fail if a parameter to its `prevent-inclusion:` does not exist
+          mkdir -p ./cleanup
```

Comment:
Could we not modify `py/pip-build-install` rather than potentially modifying multiple packages in the same way?